opentelemetry-user-events-metrics/src/exporter/mod.rs

```diff
@@ -46,411 +89,514 @@ impl Debug for MetricsExporter {
     }
 }
 
+pub(crate) fn to_nanos(time: SystemTime) -> u64 {
```

Comment:
This method need not be `pub(crate)`.

```suggestion
fn to_nanos(time: SystemTime) -> u64 {
```